query/src/main/java/org/infinispan/query/backend/IndexModificationStrategy.java

```diff
@@ -39,9 +39,13 @@ public boolean shouldModifyIndexes(FlagAffectedCommand command, InvocationContex
       @Override
```

Comment:
the java code below `ALL` talks about  `Flag.SKIP_INDEXING` but the flag is no longer checked. Is this normal ?

Reply:
The checks have been moved to `QueryInterceptor` and `TxQueryInterceptor` as the indexing should be skipped regardless of mode with this flag.